pylint/checkers/refactoring.py

```diff
@@ -29,11 +29,33 @@
 from pylint import utils as lint_utils
 from pylint.checkers import utils
 
+# sys.exit method never returns
+NEVER_RETURNING_FUNCTIONS = ('sys.exit',)
```

Comment:
@hippo91 this nees to be a comma-separated tuple that is extensible by users, here's an example https://github.com/PyCQA/pylint/blob/928e16431134a367a78b63540b7cbe5dd9967539/pylint/checkers/typecheck.py#L526. This way we allow the users to provide any functions they want